src/vl-core.js

```diff
@@ -220,15 +244,13 @@ export const vlElement = (SuperClass) => {
     }
 
     /**
-     * Definieer shadow DOM.
+     * Wijzig de (standaard) tekst van een vertaling.
      *
-     * @protected
-     * @param {Literal} html - HTML literal
-     * @return {void}
+     * @param {String} key
+     * @param {String} value
      */
-    shadow(html) {
-      this._shadow = this.attachShadow({mode: 'open'});
-      this._shadow.innerHTML = html;
+    _changeTranslation(key, value) {
```

Comment:
Nog niet aan de orde vermoed ik, maar stel dat er translations zijn voor zowel Nederlands en Engels. Als je een custom vertaling wil zetten, moet je op de een of andere manier te weten komen welke taal de gebruiker momenteel heeft zodat je de juist value kan zetten. Dit kan dan via header? footer? Of zit dit ook in dit vl.i18n object?

Reply:
Verder te bekijken wanneer de nood er is.